dogs/src/app/breed-page/breed-page.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit } from "@angular/core";
+import { ActivatedRoute, Router } from "@angular/router";
+
+import { DogService } from "../shared/api-dog.service";
+
+@Component({
+	selector: "app-breed-page",
+	templateUrl: "./breed-page.component.html",
+	styleUrls: ["./breed-page.component.css"]
+})
+export class BreedPageComponent implements OnInit {
+
+	public breedName: string = "";
+	public imageURL: any = "";
+	public subBreedName: string[] = [];
+
+	public constructor(
+		private dogService: DogService,
+		private activedRoute: ActivatedRoute,
+		private dogRoute: Router
+	) { }
+
+	public ngOnInit(): void {
+		this.getBreedImage();
+		this.getSubBreed();
+	}
+
+	private getBreedImage(): void {
```

Comment:
```suggestion
	private prepareBreedImage(): void {
```
because the method doesn't return anything